tests/simpleKVBC/TesterReplica/setup.cpp

```diff
@@ -186,6 +194,10 @@ std::unique_ptr<TestSetup> TestSetup::ParseArgs(int argc, char** argv) {
           cronEntryNumberOfExecutes = concord::util::to<std::uint32_t>(optarg);
           break;
         }
+        case 'g': {
```

Comment:
let's use 'z' as in byZantine

Reply:
Thanks @teoparvanov for this suggestion. 'z' is already used below:
{"consensus-batching-flush-period", required_argument, 0, 'z'},
So I have chosen 'g', which is the next similar sounding word.
z == pronounced as zee
g == pronounced as gee

Reply:
ok, 'g' is fine then, thanks Amitava! Closing my remark.